_posts/2024-02-06-new-ship-pathfinder.md

```diff
@@ -0,0 +1,88 @@
+---
+title: New ship pathfinder. Goodbye buoys!
+author: Kuhnovic
+---
+
+OpenTTD 14.0 comes with a new ship pathfinder which allows ships to travel great distances without the use of buoys.
+Gone are the days of long strings of buoys and convoluted ship orders.
+Using ships is now as hassle-free as the other modes of transport.
+Just add the destination and send the ship on it's way!
```

Comment:
I have absolutely no business correcting the grammar of someone who speaks three times as many languages as me, but since you mention it: `it's` is a contraction of `it is`, while the possessive is `its`. 🙂 

```suggestion
Just add the destination and send the ship on its way!
```

Reply:
You are too kind! I make this mistake all the time, and *it's* all to easy to overlook.